src/solver/modelParser/encoders.hxx

```diff
@@ -54,15 +61,15 @@ struct convert<Antares::Solver::ModelParser::ValueType>
         {
             return false;
         }
-        if (node.as<std::string>() == "FLOAT")
+        if (node.as<std::string>() == "continuous")
```

Comment:
you don't need to convert to `std::string` N times, you can convert it once and for all (l. 64, l. 68, l. 72). The compiler may optimize this away, not sure though